docs/Reference-for-template.json.md

```diff
@@ -265,7 +311,7 @@ A symbol that defines transformation of another symbol.  The value of this symbo
 |Name|Description|Mandatory|
 |---|---|---|
 |`type`|`derived`| yes |
-|`valueSource`|The name of the other symbol whose value will be used to derive this value.| yes |
+|`valueSource`|The name of the other symbol which value will be used to derive this value.| yes |
```

Comment:
I belieive which is for singular

Reply:
I think `whose` can only be applied to a person, but not sure.

Reply:
You are indeed correct.
You can disregard :)